### DISCUSSION TITLE
I dont understand whats happening

### DISCUSSION THREAD

**Discussion by wtfisfail** [community/community#46237] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Everytime I try to clone it ends up cloning an entirely different thing some skyrimse git

**Comment by jge162** [comment]
Need more context. Can’t assist with what you have provided so far. Thanks.

**Reply by wtfisfail** [reply]
So I try to clone another repo and it clones an entirely different one. I can show you in pictures. 
![Screenshot 2023-02-03 022325](https://user-images.githubusercontent.com/124333681/216538013-950c6b2e-4325-405f-bbce-c1ecb0627f7b.png)
![Screenshot 2023-02-03 022351](https://user-images.githubusercontent.com/124333681/216538058-1c49f169-be8e-4b0d-87f4-da588084eeab.png)
![Screenshot 2023-02-03 022426](https://user-images.githubusercontent.com/124333681/216538085-7a56f4e6-334d-422c-a792-7765550e0802.png)
![Screenshot 2023-02-03 022510](https://user-images.githubusercontent.com/124333681/216538106-389ce244-c991-4bb4-834a-23f8e9d7b2f6.png)

**Reply by jge162** [reply]
Try clearing your cache and restarting clone after.

**Reply by wtfisfail** [reply]
same thing happened

**Reply by jge162** [reply]
when you hit clone, double check the URL is not wrong.

**Reply by wtfisfail** [reply]
the thing is ive never tried cloning the url its cloning

**Reply by jge162** [reply]
It could be pre-filling url wrong. So double check

**Reply by wtfisfail** [reply]
like the url is empty i then copy and paste the link. in the first picture is the link im trying to get cloned the third picture shows what ends up being cloned

**Reply by jge162** [reply]
the url cannot be right, go to it online to verify

**Reply by wtfisfail** [reply]
I opened it with github desktop and it still cloned a different page

**Reply by wtfisfail** [reply]
https://user-images.githubusercontent.com/124333681/216571409-0a825a6a-0b72-4e37-93f3-8a7617939db0.mov

**Reply by wtfisfail** [reply]
https://user-images.githubusercontent.com/124333681/216572853-8accb6b1-6788-48d3-a5b3-49eff681a496.mov

**Comment by MostafaWaleed0** [comment]
1. Double-check the repository URL you are using to clone the repository, to make sure it is correct and complete.
2. Try cloning the repository from a different network or location to see if network connectivity issues are affecting the cloning process.

**Reply by wtfisfail** [reply]
https://user-images.githubusercontent.com/124333681/216572949-a545dea1-8526-406e-a02e-1e8fd6d08846.mov
